project2/styles.css

```
@@ -0,0 +1,121 @@
+* {
+    box-sizing: border-box;
+    transition: all 1s;
+}
+
+body {
+    background-color: #111111;
+    color: e0e0e0;
+    font-family: Rubik, Sans-serif;
```

The font-stack backup needs to be all lowercase, like so:
```suggestion
    font-family: Rubik, sans-serif;
```